docs/docusaurus.config.js

```diff
@@ -218,6 +210,24 @@ const config = {
       {
         fromExtensions: ["html"],
         toExtensions: ["html"],
+        createRedirects(existingPath) {
+          return [
+            existingPath.replace('/core', '/develop/advanced-concepts'),
+            existingPath.replace('/basics', '/develop/high-level-concepts'),
+            existingPath.replace('/intro', '/develop/intro'),
+            existingPath.replace('/architecture', '/integrate/architecture/'),
+            existingPath.replace('/building-apps', '/integrate/building-apps'),
+            existingPath.replace('/building-modules', '/integrate/building-modules'),
+            existingPath.replace('/tooling', '/integrate/tooling'),
+            existingPath.replace('/migrations', '/integrate/libraries'),
+            existingPath.replace('/modules', '/integrate/modules'),
+            existingPath.replace('/rfc', '/integrate/rfc'),
+            existingPath.replace('/spec', '/integrate/spec'),
+            existingPath.replace('/tooling', '/integrate/tooling'),
+            existingPath.replace('/run-node', '/user/run-node'),
+            existingPath.replace('/validate', '/user/validate')
+          ];
+        },
```

Comment:
added this in here so its uniform with the docs docusaurus.js, should be able to be deleted later